data/meetings/d2-sober-sunrise.yaml

```diff
@@ -8,11 +8,10 @@ time:
   Thursday: ["06:45"]
   Friday: ["06:45"]
 address: zoom,Online-Only,,
-note: Meeting ID:  944-241-0663    Password:  Rise@645
+note: | 
+     Meeting ID:  944-241-0663    Password:  Rise@645
     <b>Zoom:</b> <a href="https://us02web.zoom.us
```

Comment:
This link also appears to be incomplete

Reply:
The anchor was partly removed in https://github.com/area63aa/area63aa.org/commit/7f595fde0b2ce18c4b9249b95104b33c46049791. I suspect the "correct" link changed and the editor didn't know how to adjust this.